internal/jujuapi/controller_test.go

```diff
@@ -52,22 +52,6 @@ func (s *controllerSuite) TestControllerConfigSetNotSupported(c *gc.C) {
 	c.Assert(jujuparams.IsCodeNotSupported(err), gc.Equals, true)
 }
 
-func (s *controllerSuite) TestModelConfig(c *gc.C) {
-	conn := s.open(c, nil, "test")
-	defer conn.Close()
-	client := controllerapi.NewClient(conn)
-	_, err := client.ModelConfig()
```

Comment:
The model config client seems to have been removed and since we were testing that it is not supported, I have removed this test until I can figure out where the client should be coming from (or I can link the PR where it was removed).

Reply:
Removed in https://github.com/juju/juju/pull/17732